h2o-py/tests/testdir_apis/H2O_Module/pyunit_h2oshow_progress.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+from tests import pyunit_utils
+import h2o
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+
+def h2oshow_progress():
+    """
+    Python API test: h2o.show_progress()
+
+    Command is verified by eyeballing the pyunit test output file and make sure the progress bars are there.
+    Here, we will assume the command runs well if there is no error message.
+    """
+    try:
+        h2o.show_progress()   # true by default.
```

Comment:
maybe do a `h2o.no_progress()` first, then `h2o.show_progress()`, and then same trick as above to verify that progress bar properly appears